uvicorn/protocols/utils.py

```diff
@@ -103,15 +130,19 @@ def get_tls_info(transport: asyncio.Transport) -> dict[object, object]:
         for rdn in peercert["subject"]:
             rdn_strings.append(
                 "+".join(
-                    [f"{RDNS_MAPPING[entry[0]]} = {entry[1]}" for entry in reversed(rdn) if entry[0] in RDNS_MAPPING]
+                    [
+                        f"{RDNS_MAPPING[entry[0]]}={escape_dn_chars(entry[1])}"
+                        for entry in reversed(rdn)
+                        if entry[0] in RDNS_MAPPING
+                    ]
                 )
             )
 
         ssl_info["client_cert_chain"] = [ssl.DER_cert_to_PEM_cert(ssl_object.getpeercert(binary_form=True))]
-        ssl_info["client_cert_name"] = ", ".join(rdn_strings) if rdn_strings else ""
+        ssl_info["client_cert_name"] = ",".join(rdn_strings) if rdn_strings else ""
         ssl_info["tls_version"] = (
             TLS_VERSION_MAP[ssl_object.version()] if ssl_object.version() in TLS_VERSION_MAP else None
         )
-        ssl_info["cipher_suite"] = list(ssl_object.cipher())
+        ssl_info["cipher_suite"] = TLS_CIPHER_SUITES[ssl_object.cipher()[0]]
```

Comment:
If the choosen cipher is not in the table, perhaps we should return None?

`TLS_CIPHER_SUITES.get(ssl_object.cipher()[0], None)`